nativelib/src/main/scala/scala/scalanative/native/stdlib.scala

```diff
@@ -2,8 +2,8 @@ package scala.scalanative
 package native
 
 @extern object stdlib {
-  var __stderrp: Ptr[_] = extern
-  var __stdoutp: Ptr[_] = extern
+  var stderr: Ptr[_] = extern
+  var stdout: Ptr[_] = extern
```

Comment:
Lets keep this PR for the docs only, and continue with discussion of this on the issue I've mentioned above.